integration/tests/crud/testdata/scale-nodegroups.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: eksctl.io/v1alpha5
+kind: ClusterConfig
+
+# name is generated
+metadata:
+  name: <generated>
+
+nodeGroups:
+  - name: n1
+    desiredCapacity: 5
+    maxSize: 5
+    minSize: 5
+
+managedNodeGroups:
+  - name: ng-0
```

Comment:
`ng-0`, `n1` :ahhhhh:

Reply:
the names of the nodegroups created in the `crud` suite predate this PR but I'm happy to change them for consistency 👍

Reply:
you're right, I'll change the ng names in the `crud` package to something more consistent - what do you think of the following:
```
	const (
		initNG = "ng-1"
		testNG = "ng-2"

		unmNG1 = "unm-ng-1"
		unmNG2 = "unm-ng-2"
	)
```

Reply:
I think we just do `mng-1` and `ng-1` in other tests for managed vs unmanaged, but its still not great either